source/documentation/administration_guide/topics/Configuring_the_Red_Hat_Virtualization_Manager_to_Send_SNMP_Traps.adoc

```diff
@@ -1,11 +1,21 @@
 [[Configuring_the_Red_Hat_Enterprise_Virtualization_Manager_to_Send_SNMP_Traps]]
-=== Configuring the {virt-product-fullname} {engine-name} to Send SNMP Traps
+= Configuring the {virt-product-fullname} {engine-name} to Send SNMP Traps
 
-Configure your {virt-product-fullname} {engine-name} to send Simple Network Management Protocol traps to one or more external SNMP managers. SNMP traps contain system event information; they are used to monitor your {virt-product-fullname} environment. The number and type of traps sent to the SNMP manager can be defined within the {virt-product-fullname} {engine-name}.
+Configure your {virt-product-fullname} {engine-name} to send Simple Network Management Protocol (SNMP) traps to one or more external SNMP managers. SNMP traps contain system event information; they are used to monitor your {virt-product-fullname} environment. The number and type of traps sent to the SNMP manager can be defined within the {virt-product-fullname} {engine-name}.
 
-This procedure assumes that you have configured one or more external SNMP managers to receive traps, and that you have the following details:
+{virt-product-fullname} supports SNMP version 2 and version 3. SNMP version 3 supports the following security levels:
 
-* The IP addresses or fully qualified domain names of machines that will act as SNMP managers. Optionally, determine the port through which {engine-name} receives trap notifications; by default, this is UDP port 162.
+*NoAuthNoPriv*:: SNMP traps are sent without any authorization or privacy.
+*AuthNoPriv*:: SNMP traps are sent with password authorization but no privacy.
+*AuthPriv*:: SNMP traps are sent with password authorization and privacy.
+
+.Prerequisites
+
+* One or more external SNMP managers is configured to receive traps.
```

Comment:
One or more external SNMP managers **are** configured to receive traps

Reply:
other that this grammatical suggestion, 
/lgtm

Reply:
Fixed.